### ISSUE TITLE
How do you register/observe changes to a particular Vehicle parameter

### ISSUE THREAD

**Issue by hamishwillee** [diydrones/dronekit-python#89]
class Parameters implements HasObservers interface, and it should therefore be possible to register to observe parameter changes (this is also indicated by the original documents). 

I tried this, but while the code will run, the observer is not called (possibly because THR_MIN is not an attribute):
```
v.add_attribute_observer('THR_MIN', thin_callback)
```
I tried this, but while the fails with some kind of not-supported error
```
v.parameters.add_attribute_observer('THR_MIN', thin_callback)
```

How is this done?

**Comment by geeksville** [comment]
This is meant to be supported and if it doesn't work it would be easy to add.

**Comment by hamishwillee** [comment]
Thanks @geeksville

@mrpollo I think "we" should add this. The question is, will it be done in a timely manner, or should I update the documentation now to reflect that it is not possible right now?

**Comment by mrpollo** [comment]
update docs to reflect current state of API and please create a new issue assigned to me to add this feature

@geeksville thanks!

**Comment by hamishwillee** [comment]
OK, #107 is the issue for the missing functionality. Am assigning this to me to fix the problem.

**Comment by hamishwillee** [comment]
Documentation fixed by #108. Closing.
